.golangci.yml

```diff
@@ -82,6 +82,9 @@ linters-settings:
       - sync/atomic: "Use go.uber.org/atomic instead; see docs/dev/atomics.md"
       - io/ioutil: "Deprecated since Go 1.16. Use package io or os instead."
       - github.com/golang/glog: "Crashes Windows nanoserver and significantly delays Agent startup on Windows Domain Controllers."
+      # feel free to remove if you really need it as opposed to testify/assert
```

Comment:
Do you mean removing it at the linted file scope ?

If that's the case could you add the comment line one should use to ignore the linter for this file ?